CRM/Upgrade/3.2.alpha1.msg_template/message_templates/event_offline_receipt_text.tpl

```diff
@@ -20,9 +19,7 @@ a link to a web page where you can complete your registration.{/ts}
 {ts}Your registration has been submitted.{/ts}
 
 {if $isPrimary}
-{ts}Once your registration has been reviewed, you will receive
-an email with a link to a web page where you can complete the
-registration process.{/ts}
+{ts}Once your registration has been reviewed, you will receive an email with a link to a web page where you can complete the registration process.{/ts}
 
```

Comment:
Traditionally, one was expected to wrap plain-text emails at 78 characters. However, now there seem to be [differing](https://www.campaignmonitor.com/blog/post/3780/removing-the-60-character-line-limit-on-plain-text-email) [opinions](http://blog.aweber.com/email-marketing/usable-plain-text-emails.htm).

The most interesting article, IMHO, is https://mathiasbynens.be/notes/gmail-plain-text which suggests that it's OK to ship unwrapped plain-text -- as long as you set an extra headers. However, I don't think we're setting that header.